GameData/ROEngines/PartConfigs/Raptor_RE.cfg

```diff
@@ -73,12 +73,12 @@ PART
 	
 	MODULE
 	{
-	    name = ModuleGimbal
-	    gimbalTransformName = gimbal
+		name = ModuleGimbal
+		gimbalTransformName = gimbal
 		gimbalRange = 3.5
- 	    gimbalResponseSpeed = 15
- 	    useGimbalResponseSpeed = true
-    }
+		gimbalResponseSpeed = 15
+		useGimbalResponseSpeed = true
```

Comment:
Is it correct to set the gimbal parameters here instead of in the RO engine config?

Reply:
https://github.com/KSP-RO/RealismOverhaul/pull/2351#issuecomment-678842166
> [kirk] EngineConfig should only set GimbalRange. Everything else should be set in the part itself

Reply:
I read that as overriding any gimbal settings should be done in the Post pass, not the original part declaration.  I also read that as how to override for single axis gimbals, which will be part specific.  But a generic, "unrestricted" (?) gimbal with just a response speed and the flag to use it is set in the RO engine configs just about everywhere.  For consistency, does that need to be in the part or still in the RO engine config?